tritium-proxy/src/main/java/com/palantir/tritium/proxy/Proxies.java

```diff
@@ -58,41 +56,26 @@ private Proxies() {
      *
      * @param iface primary interface class
      * @param delegateClass delegate class
-     * @param additionalInterfaces additional interfaces
      * @return the set of interfaces for the specified classes
      * @throws IllegalArgumentException if the specified interface class is not an interface
      */
     static Class<?>[] interfaces(Class<?> iface,
-                                 Class<?> delegateClass,
-                                 Collection<Class<?>> additionalInterfaces) {
+                                 Class<?> delegateClass) {
```

Comment:
not directly related, but I removed the package-private unused `additionalInterfaces` parameter here